site/src/Librecores/ProjectRepoBundle/Resources/views/Project/settings.html.twig

```diff
@@ -123,6 +173,46 @@
       </div>
     </div>
   </div>
+
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Delete</h3>
+    </div>
+    <div class="panel-button" id="delete-panel-button">
+      <br>
+      <a href="#" id="new-delete-button" class="btn btn-default" data-toggle="modal" data-target="#delete-modal"><strong>Delete this project</strong></a>
```

Comment:
what is a "new delete button"? Please find a more descriptive name for this `id` attribute.

Reply:
Changed the id to delete-panel-button